tailwind.config.ts

```
@@ -1,18 +1,63 @@
+  			}
+  		},
+  		borderRadius: {
+  			lg: 'var(--radius)',
+  			md: 'calc(var(--radius) - 2px)',
+  			sm: 'calc(var(--radius) - 4px)'
+  		}
+  	}
   },
-  plugins: [],
-} satisfies Config;
+    plugins: [require("tailwindcss-animate")]
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)